Sample/Tickets/Tickets.Api.Tests/Reservations/CreateTentativeReservationTests.cs

```diff
@@ -0,0 +1,158 @@
+using System;
+using System.Linq;
+using System.Net;
+using System.Net.Http;
+using System.Threading.Tasks;
+using Core.Testing;
+using FluentAssertions;
+using Marten.Pagination;
+using Tickets.Api.Requests;
+using Tickets.Api.Responses;
+using Tickets.Api.Tests.Core;
+using Tickets.Reservations;
+using Tickets.Reservations.Events;
+using Tickets.Reservations.Projections;
+using Xunit;
+
+namespace Tickets.Api.Tests.Reservations
+{
+    public class CreateTentativeReservationFixture: ApiFixture<Startup>
+    {
+        protected override string ApiUrl { get; } = "/api/Reservations";
+
+        public readonly Guid SeatId = Guid.NewGuid();
+
+        public HttpResponseMessage CommandResponse;
+
+        public override async Task InitializeAsync()
+        {
+            // send create command
+            CommandResponse = await PostAsync(new CreateTentativeReservationRequest {SeatId = SeatId});
+        }
+    }
+
+    public class CreateTentativeReservationTests: IClassFixture<CreateTentativeReservationFixture>
+    {
+        private readonly CreateTentativeReservationFixture fixture;
+
+        public CreateTentativeReservationTests(CreateTentativeReservationFixture fixture)
+        {
+            this.fixture = fixture;
+        }
+
+        [Fact]
+        [Trait("Category", "Exercise")]
+        public async Task CreateCommand_ShouldReturn_CreatedStatus_With_ReservationId()
+        {
+            var commandResponse = fixture.CommandResponse;
```

Comment:
It would be nice to add a message here in the case of error. ( from the Test result you can't get any human-readable information what is the problem)
```suggestion
            var commandResponse = fixture.CommandResponse;
            var jsonContent = await commandResponse .Content.ReadAsStringAsync();
            Console.WriteLine(jsonContent);
```